proto/api/v1/administration_events_service.proto

```diff
@@ -90,16 +90,16 @@ message AdministrationEventsFilter {
     google.protobuf.Timestamp until = 2;
 
     // Matches events from a specific domain.
-    string domain                   = 3;
+    repeated string domain                   = 3;
```

Comment:
Can you fix the `=` alignment? Btw does GoLand have a formatter for this? Unfortunately I didn't find the formatter we are using here. https://github.com/bufbuild/buf formats it completely differently...

Reply:
Yeah the GoLand formatter ain't it, I'm not even sure it follows a specific formatter (at the very least I don't know which to be honest).